managed/models/database.go

```diff
@@ -838,6 +838,50 @@ var databaseSchema = [][]string{
 		FROM services
         WHERE service_type = 'mongodb';`,
 	},
+	79: {
+		`CREATE TABLE available_tips (
+			id SERIAL PRIMARY KEY,
+			type VARCHAR NOT NULL,
+		
+			created_at TIMESTAMP NOT NULL,
+			updated_at TIMESTAMP NOT NULL
+		);
+		
+		CREATE TABLE system_tips (
+			 id SERIAL PRIMARY KEY,
+			 is_completed BOOLEAN NOT NULL,
+		
+			 created_at TIMESTAMP NOT NULL,
+			 updated_at TIMESTAMP NOT NULL
```

Comment:
available_tips - can be an enum.

why `system_tips` are stored? it is something we check every time, we don't need to store that

Reply:
because when user disconnects agent/database if we don't store the status of the tip it will be again uncompleted.

To avoid this we store the status when user connects/disconnects the agent or external db.